app/policies/activity_policy.rb

```diff
@@ -7,12 +7,16 @@ def show?
     user.administrator? || user.fund_manager?
   end
 
+  def edit?
```

Comment:
If we are changing the logic so that anyone who can create an activity can also edit it then I think we could refactor this method out since we have:

```
def update?
  create?
end
```

and the [application policy would do handle the edit? logic for us automatically](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/app/policies/application_policy.rb#L31).

Reply:
Nice one! I wondered where edit was coming from!